test/spec.yml

```diff
@@ -467,6 +478,22 @@
         err: "Error: getaddrinfo ENOTFOUND fake.run.binaris.invalid. fake.run.binaris.invalid.:443"
         exit: 1
 
+- test: Test feedback empty message (bad-path)
+  setup:
+    - export FEEDBACK_EMAIL=test@binaris.com
+  steps:
+    -   in: bn feedback "$FEEDBACK_EMAIL" ""
+        err: Not a valid message.
+        exit: 1
+
+- test: Test feedback empty email (bad-path)
```

Comment:
since currently a third option is not allowed. i'd add another bad path of `bn feedback foo@binaris.com this and that`